gramps/gen/utils/docgen/odstab.py

```diff
@@ -335,7 +335,8 @@ def open(self,filename):
 
         try:
             self.content_xml = tempfile.mktemp()
-            self.f = open(self.content_xml,"wb")
+            with open(self.content_xml,"wb") as self.f:
```

Comment:
I believe I have this one correct, as this line only test that it can open/create the file and then on line [344 ](https://github.com/gramps-project/gramps/blob/1a05aadca7f78f0ec34cda74c087e38bb15c2318/gramps/gen/utils/docgen/odstab.py#L336-L344)it actually opens the file in the original code with:
`self.f = open(self.content_xml,"w", encoding='utf-8')`